README.md

```diff
@@ -71,6 +71,41 @@ You can use `eth-staking-smith` as follows to convert your address:
 
 Note that --validator-index and --validator-start-index are two distinct parameter, the former being index of validator on Beacon chain, and the latter is the index of validator private key derived from the seed
 
+## Exporting CLI standard output into common keystores folder format
+
+Most validator clients recognize the keystore folder format,
+produced by upstream Python deposit CLI. While `eth-staking-smith` outputs
+all validator data into standard output, allowing for better security in
+enterprise setups, for small and individual stakers this is not convenient,
+as they need to be able to import validator keys directly into validator client.
+
+To address such needs, `eth-staking-smith` provides convenience Python3 script
```

Comment:
why does it need a separate python script?

Reply:
The solution with separate script is because eth-staking-smith itself does not work with filesystem as it stated on the readme to increase security, so all filesystem operations to make it compatible with common keystore folder are done in a separate convenience script.